### PR TITLE
Add intro slide

### PR DESCRIPTION
**Pull request by stuebingerb**
Adds a hacky intro slide that shows the current date (as YYYY-MM-DD) along with a title provided via query parameter.

Use locally, e.g. as
```
file:///path/to/jug-in.talks/jug-intro.html?title=Foo\nBar#/
```

`\n` in the parameter value will result in a line break and whitespace will be preserved. Empty or missing title will default to "Java-Treff".

Sample/default view:
image

### CODE REVIEW

**Review by heubeck — APPROVED** [review]
:guitar:

File: jug-intro.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><title>JUG Ingolstadt e.V.</title><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui" name="viewport"><link href="node_modules/reveal.js/css/reveal.css" rel="stylesheet"><link rel="stylesheet" href="node_modules/reveal.js/css/theme/white.css" id="theme"><link href="node_modules/reveal.js/lib/css/zenburn.css" rel="stylesheet"><!--[if lt IE 9]><script src="node_modules/reveal.js/lib/js/html5shiv.js"></script><![endif]--><link rel="stylesheet" href="talks/_theme/theme.css"></head><body>
```
Comment (beiertu-mms) [inline jug-intro.html]: this file is generated? or why it's in the `.gitignore`?

Comment (stuebingerb) [reply jug-intro.html]: Usually html files are generated from the asciidoc sources and therefore ignored. The intro slide is manually crafted, so it needs to be excluded from `.gitignore`.

Comment (beiertu-mms) [reply jug-intro.html]: I see :)
can you format the file as well? 😁

Comment (stuebingerb) [reply jug-intro.html]: _Just_ for you.

**Review by beiertu-mms — APPROVED** [review]

**Pull request merged**